<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>1000</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFeature::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::DockWidgetArea::NoDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Drawing Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>417</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="ColorLabel">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="redButton">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>red</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="blueButton">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>blue</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="greenButton">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>green</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="yellowButton">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>yellow</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="thicknessLabel">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Thickness</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="thicknessSlider">
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="styleLabel">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Style</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="solidButton">
           <property name="text">
            <string>Solid Line</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="dashButton">
           <property name="text">
            <string>Dash Line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="dotButton">
           <property name="text">
            <string>Dot Line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="shapeLabel">
           <property name="text">
            <string>Shape</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="lineButton">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Line</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rectangleButton">
           <property name="text">
            <string>Rectangle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ellipseButton">
           <property name="text">
            <string>Ellipse</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
